src/wast-parser.y

```diff
@@ -149,6 +144,65 @@ class BinaryErrorHandlerModule : public ErrorHandler {
   WastParser* parser_;
 };
 
+template <typename OutputIter>
+void UnescapeString(string_view text, OutputIter dest) {
```

Comment:
Perhaps better named 'RemoveEscapes' (the string can be dropped since it is implied by the first argument).